RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -365,7 +380,7 @@ public void exportProgress(int progress){
         mPd.show();
     }
 
-    public void zipProgress(int progress){
+    public void zipProgress(int progress) {
```

Comment:
Same as above, I think a better name for this is `showZipProgress()`

Also, I don't see `progress` is being used inside the method.